testslide/__init__.py

```diff
@@ -207,6 +207,12 @@ class UnexpectedSuccess(Exception):
     """
 
 
+class SlowCallback(Exception):
```

Comment:
Docs refer to this as "Even Loop Lock Up", perhaps we could keep the same naming in both places.

Reply:
I don't see where in the docs it's referenced as "event loop lock up", but I see
> Callbacks taking longer than 100ms are logged. The loop.slow_callback_duration attribute can be used to set the minimum execution duration in seconds that is considered “slow”

In the debugging section of https://docs.python.org/3/library/asyncio-dev.html

Reply:
Oh, I'm sorry, I meant [TestSlide's](https://github.com/facebookincubator/TestSlide/blob/master/docs/testslide_dsl/async_support/index.rst#even-loop-lock-up) documentation which I added yesterday. I agree with you that using Python's naming is better, so please feel fere to amend this at TestSlide's docs (also check the example output, the exception / message needs updating as well).